sentences/de/binary_sensor_HassGetState.yaml

```
@@ -132,3 +132,42 @@ intents:
         slots:
           domain: binary_sensor
           device_class: carbon_monoxide
+
+      # Connectivity
+      - sentences:
+          - "(ist|sind) <name> [<area>] {bs_connectivity_states:state}"
```

```suggestion
          - "ist <name> [<area>] {bs_connectivity_states:state}"
          - "ist <area> <name> {bs_connectivity_states:state}"
```

_4 possible test sentences_